### ISSUE TITLE
The "Add x" pages are missing local task tabs.

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#1507]
```admin/structure/menu/manage/user-menu``` (by the way shouldn't this be ```admin/structure/menu/manage/account-menu``` instead?):

![backdrop-menu-edit_links](https://cloud.githubusercontent.com/assets/2423362/12221770/c43a425a-b7fc-11e5-9bfa-398259a61318.png)

So the breadcrumb, page title, local task is:

Home > Administration > Structure > Menus > Account menu > Edit links

```admin/structure/menu/manage/user-menu/add``` (no active tab):

![backdrop-menu-add_link](https://cloud.githubusercontent.com/assets/2423362/12221788/343445c4-b7fd-11e5-98a0-06c1956cf760.png)

Here the breadcrumb, page title, local task is:

Home > Administration > Structure > Menus > Account menu > Account menu > ???

...should be:

Home > Administration > Structure > Menus > Account menu > Add link

**Comment by quicksketch** [comment]
Thanks for opening this issue @klonos. I think you've identified another UX issue.

> Home > Administration > Structure > Menus > Account menu > Add link

It is still the case in Backdrop (as it has been in Drupal forever) that the breadcrumb does not contain the current page. So in the case of all pages here (Add link, Edit links, Configure) the breadcrumb is the same for all of them at `Home > Administration > Structure > Menus > Account menu`.

So in the case of breadcrumbs, everything is as it was at least *intended* to function. If you have suggestions on changing that, we can do it in a different issue.

But the page is missing any indicator of the current path (with both tabs inactive) indeed seems problematic for UX. I'm not sure what the solution is here. In D6, we didn't have "Local actions" (like the "Add link" link). Everything was just a tab, so we didn't have this particular problem.

Personally, I like the idea behind local actions (the list of links) being separate from local tasks (the list of tabs) because it separates manipulating an item itself from creating a new child item. And I like the appearance of it, which I'm pretty sure is the reason why they came into existence in the first place when Acquia hired Mark Boulton.

**Comment by klonos** [comment]
Fair enough, but we do need some indication as to where the user currently is. How about we change the page title in this case to be "Add link" instead of "Account menu" as it is currently (duplicating the last part of the breadcrumb)?

**Comment by klonos** [comment]
...I mean in general add the local action as page title if that's where the user is.

**Comment by quicksketch** [comment]
> Fair enough, but we do need some indication as to where the user currently is.

Yes, totally agreed! Sorry if I was sounding defensive of the current pattern. There's definitely a problem here.

> ...I mean in general add the local action as page title if that's where the user is.

This sounds good to me. We could also repeat the title, such as "Add Link - Account menu" (or some variation of). We could also do a more specific "Add link to Account menu" phrase, but we'd likely have to specify each of those messages individually, as "to" is not always the right preposition.

Or we could just stick with the action link as the title. They're all good options, better than the current situation.

**Comment by klonos** [comment]
I think perhaps "Account menu: Add link". It's what we have chosen to go with in the search results in the admin bar. Consistency! :stuck_out_tongue_winking_eye:
